packages/site/src/components/Navigation/TopNav.tsx

```diff
@@ -57,7 +57,7 @@ function ExternalOrInternalLink({
 function NavItem({ item }: { item: SiteNavItem }) {
   if (!('children' in item)) {
     return (
-      <div className="relative grow-0 inline-block mx-2">
+      <div className="relative inline-block mx-2 grow-0">
```

Comment:
What vscode extension do you use to order tailwind classes? (I am assuming?)

I was looking into/thinking about this today!

Reply:
I use [Headwind](https://marketplace.visualstudio.com/items?itemName=heybourn.headwind).

Reply:
Doing this for the whole project in #135. I will merge after this PR goes in so there are no conflicts.